packages/apps/plugins/plugin-map/src/components/MapControl.tsx

```diff
@@ -21,29 +22,30 @@ export type Marker = {
 
 const defaults = {
   center: { lat: 37.970833, lng: 23.72611 } as LatLngExpression,
-  zoom: 13,
+  zoom: 1,
 };
 
 /**
  * https://react-leaflet.js.org/docs/api-map
  */
-export const MapControl: FC<{ zoom?: number; center?: LatLngExpression; markers?: Marker[] }> = ({
-  zoom = defaults.zoom,
-  center = defaults.center,
```

Comment:
Remove unused API options from `MapControl`, this way we don't have to deal with cases like whether provided `zoom` and `center` arguments or zooming in on the markers takes precedence. We can always add them back later if desired.